content/en/docs/tasks/configure-pod-container/migrate-from-psp.md

```diff
@@ -19,30 +19,552 @@ admission controller. This can be done effectively using a combination of dry-ru
+   identified, the original Pod or PodTemplate should be updated with the desired configuration.
+   As a reminder, the fields to review are:
+   - `.spec.containers[*].securityContext.allowPrivilegeEscalation`
+   - `.spec.initContainers[*].securityContext.allowPrivilegeEscalation`
+   - `.spec.runtimeClassName`
+   - `.spec.securityContext.seccompProfile`
+   - `.metadata.annotations['container.apparmor.security.beta.kubernetes.io/*']` (replace * with each container name)
+   - On containers, under `.spec.containers[*]` and `.spec.initContainers[*]`:
+       - `.securityContext.allowPrivilegeEscalation`
+       - `.securityContext.seccompProfile`
+       - `.securityContext.capabilities.add`
+3. Create the new PodSecurityPolicies. If any Roles or ClusterRoles are granting `use` on all PSPs
+   this could cause the new PSPs to be used instead of their mutating counter-parts.
+4. Update your authorization to grant access to the new PSPs. In RBAC this means updating any Roles
+   or ClusterRoles that grant the `use` permision on the original PSP to also grant it to the
+   updated PSP.
+5. Verify: after some soak time, rerun the command from step 1 to see if any pods are still using
+   the original profiles. If the pods haven't been updated since the new policies were rolled out,
+   they may need to be cycled to be verified against the new PSPs.
+6. (optional) Once you have verified that the original PSPs are no longer in use, you can delete
+   them.
+
+## 3. Update Namespaces
+
+The following steps will need to be performed on every namespace in the cluster. Commands referenced
+in these steps use the `$NAMESPACE` variable to refer to the namespace being updated.
+
+### 3.a. Identify an appropriate Pod Security level
+
+Start reviewing the [Pod Security Standards](/docs/concepts/security/pod-security-standards/) and
+familiarizing yourself with the 3 different profile levels.
+
+There are several ways to choose a Pod Security level for your namespace:
+
+1. **By security requirements for the namespace** - if you are familiar with the expected access
+   level for the namespace, you can choose an appropriate level based on those requirements, similar
+   to how one might approach this on a new cluster.
+2. **By existing Pod Security Policies** - Using the [table below](#psp-fields) you can map each
+   PSP to a Pod Security Standard level. If your PSPs aren't based on the PodSecurityStandards, you
+   may need to decide between choosing a level that is at least as permissive as the PSP, and a
+   level that is at least as restrictive. You can see which PSPs are in use for pods in a given
+   namespace with this command:
+   ```sh
+   kubectl get pods -n $NAMESPACE -o jsonpath="{.items[*].metadata.annotations.kubernetes\.io\/psp}" | tr " " "\n" | sort -u
+   ```
+3. **By existing pods** - Using the strategies under [Verify the Pod Security
+   level](#verify-pss-level), you can test out both the Baseline and Restricted levels to see
```

Comment:
Do we want to use 'Baseline' or 'baseline'? Please make them consistent through out the page.